staging/src/k8s.io/apiextensions-apiserver/test/integration/fixtures/server.go

```diff
@@ -90,7 +90,7 @@ users:
 		"--kubeconfig", fakeKubeConfig.Name(),
 		// disable admission and filters that require talking to kube-apiserver
 		"--enable-priority-and-fairness=false",
-		"--disable-admission-plugins", "NamespaceLifecycle,MutatingAdmissionWebhook,ValidatingAdmissionWebhook"},
+		"--disable-admission-plugins", "NamespaceLifecycle,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ValidatingAdmissionPolicy"},
```

Comment:
We are still disabling VAP plugin here. Could you confirm that is what we want? Thank you.

Reply:
good catch, I'd expect this to "just work" now with the admission init move

Reply:
Yes, refer the above comments. This test setting is specific to extended apiserver with admission and filter disabled. I guess it makes sense to keep VAP plugin the same setting as others here.

Reply:
ah, ok... if we're not giving it a connection to kube-apiserver then it won't work :)

I'm fine with this either way... if it works with VAP, great. If it doesn't (like all the other admission plugins that require talking to kube-apiserver in this specific test), I'm ok with it being disabled in this specific test.